src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -2601,6 +2607,8 @@ public partial class Exception : System.Runtime.Serialization.ISerializable
         public virtual string StackTrace { get { return default(string); } }
         public System.Reflection.MethodBase TargetSite { get { return default(System.Reflection.MethodBase); } }
         protected event System.EventHandler<System.Runtime.Serialization.SafeSerializationEventArgs> SerializeObjectState { add { } remove { } }
+        [System.Runtime.CompilerServices.FriendAccessAllowedAttribute]
+        internal void AddExceptionDataForRestrictedErrorInfo(string restrictedError, string restrictedErrorReference, string restrictedCapabilitySid, object restrictedErrorObject, bool hasrestrictedLanguageErrorObject = false) { }
```

Comment:
Yes this is worth tracking as an issue but we still need to use caution when doing this switch because we have to ensure our libraries work on existing platforms as well which don't have the System.Private.CoreLib assembly. For example: if we wanted a library to work for UWP F5 on CoreCLR that we shipped at RTM last summer it cannot have a reference to S.P.CoreLib it will still have to reference mscorlib. So we may need to have both.